x/profiles/simulation/genesis.go

```diff
@@ -15,7 +15,7 @@ import (
 
 // RandomizedGenState generates a random GenesisState for profile
 func RandomizedGenState(simsState *module.SimulationState) {
-	profilesNumber := simsState.Rand.Intn(len(simsState.Accounts) - 10)
+	profilesNumber := simsState.Rand.Intn(len(simsState.Accounts) - 5)
```

Comment:
Why was this changed? @dadamu

Reply:
The random of profilesNumber will be exactly 1 so that the `simsState.Rand.Intn(profilesNumber/2)` in the following codes occurs the panic which is `invalid argument to Intn`. So I simply changed the number to reduce.